lib/credo_language_server/code_actions/module_doc.ex

```diff
@@ -0,0 +1,24 @@
+defmodule CredoLanguageServer.CodeActions.ModuleDoc do
+  def actions(uri, diagnostic) do
+    position = %GenLSP.Structures.Position{
+      line: diagnostic.range.start.line + 1,
+      character: 0
+    }
+
+    [
+      %GenLSP.Structures.CodeAction{
+        title: "Add \"@moduledoc false\"",
+        edit: %GenLSP.Structures.WorkspaceEdit{
+          changes: %{
+            uri => [
+              %GenLSP.Structures.TextEdit{
+                new_text: "  @moduledoc false\n",
```

Comment:
I think this is fine for now, and matches the behavior of the existing action, but ti would be nice if we could make the new text follow the same amount of indent as the surrounding text, that would be nice.

Reply:
Yeah, I agree. Maybe it breaks with nested modules. I'm gonna try a better approach